docs/fwupd.conf.md

```diff
@@ -315,6 +315,34 @@ The `[thunderbolt]` section can contain the following parameters:
 
   Forces delaying activation until shutdown/logout/reboot.
 
+## TEST PARAMETERS
+
+The `[test]` section can contain the following parameters:
+
+**DecompressDelay={{test_DecompressDelay}}**
+
+  Delay in milliseconds to use when decompressing the test device.
+
+**WriteSupported={{test_Supported}}**
+
+  If the device write is supported.
```

Comment:
maybe best to explain the expected behavior when this is marked unsupported

Reply:
Done.